tests/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixerTest.php

```diff
@@ -97,11 +97,10 @@ private function test123(){
 EOF;
         $this->doTest($this->removeLinesFromString($template, $lineNumberRemoved), $template);
 
-        if (null === $config) {
-            $this->fixer->configure(array());
-        } else {
-            $this->fixer->configure(array('tokens' => $config));
-        }
+        null === $config
```

Comment:
I think what @keradus meant is that the initial goal of such ternary expression is to choose between two values (no side effect, we compute something and returns it), while here we choose between two actions.

IMO regular `if/else` is more suitable for choosing between actions, at least for readability.